webview-ui/src/config/platform-configs.json

```
@@ -2,11 +2,13 @@
 	"vscode": {
 		"messageEncoding": "none",
 		"showNavbar": false,
-		"postMessageHandler": "vscode"
+		"postMessageHandler": "vscode",
+		"togglePlanActKeys": "Meta+Shift+a"
```

Should we nest it inside the keybindings field as I can imagine there will be more keybindings we will need to set in the future?

```suggestion
		"keybindings": {
				"togglePlanAct": [""meta", "shift", "a"]
		}
```

---

That's a good idea, I can add it in a second PR.